android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -77,47 +79,36 @@
 	private boolean isTruncatingText = false;
 	private boolean disableChangeEvent = false;
 
-	protected TiEditText tv;
-	
-	public class TiEditText extends EditText 
+	protected EditText tv;
+
+	public TiUIText(final TiViewProxy proxy, boolean field)
 	{
-		public TiEditText(Context context) 
-		{
-			super(context);
-		}
+		super(proxy);
+		Log.d(TAG, "Creating a text field", Log.DEBUG_MODE);
 		
-		/** 
-		 * Check whether the called view is a text editor, in which case it would make sense to 
-		 * automatically display a soft input window for it.
-		 */
-		@Override
-		public boolean onCheckIsTextEditor () {
```

Comment:
Because we're no longer extending EditText, thus we're not overriding any of its methods. Instead, we're instantiating an EditText from XML and calling methods on the instance. That's the only documented/official way for Appcompat to generate the properly styled widgets for all API versions.